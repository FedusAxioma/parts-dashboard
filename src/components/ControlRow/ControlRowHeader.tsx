import ControlRow from './ControlRow';
import React from 'react';

const ControlRowHeader: React.FC = () => {
	return (
		<ControlRow name={'Control'} deviation={'Dev'} deviationOutTotal={'Dev Out Tol'} />
	);
};

export default ControlRowHeader;
