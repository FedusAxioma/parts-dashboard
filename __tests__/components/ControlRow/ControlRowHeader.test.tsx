import ControlRowHeader from '../../../src/components/ControlRow/ControlRowHeader';
import { createRenderer } from 'react-test-renderer/shallow';
import React from 'react';

describe('App', () => {
	const shallow = createRenderer();

	test('renders component', () => {
		shallow.render(<ControlRowHeader />);

		expect(shallow.getRenderOutput()).toMatchSnapshot();
	});
});
