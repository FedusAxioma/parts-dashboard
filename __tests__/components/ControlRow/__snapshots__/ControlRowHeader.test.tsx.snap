// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ControlRowHeader renders component 1`] = `
<div>
  <UNDEFINED
    deviation="Dev"
    deviationOutTotal="Dev Out Tol"
    name="Control"
  />
</div>
`;
